subsys/bluetooth/host/gatt.c

```diff
@@ -1892,6 +1892,52 @@ static u8_t disconnected_cb(const struct bt_gatt_attr *attr, void *user_data)
 	return BT_GATT_ITER_CONTINUE;
 }
 
+bool bt_gatt_subscribed(struct bt_conn *conn,
+			const struct bt_gatt_attr *attr, u16_t ccc_value)
+{
+	const struct _bt_gatt_ccc *ccc;
+
+	if (!conn || conn->state != BT_CONN_CONNECTED) {
```

Comment:
Is it really valid to pass NULL to this API? Seems like unnecessary code to me, meaning either the check should b e completely removed or then it should be converted to an assert (however I don't think we have a practice of doing that for conn pointers).

Reply:
It seem so be a practice in this file to assert on conn pointers. So will do that for this function as well.